src/Button/Button.tsx

```diff
@@ -519,7 +519,7 @@ export const Button = React.forwardRef<HTMLElement, Props>(
                   {icon && (
                     <ButtonIcon
                       iconSize={iconSize}
-                      className={css({ margin: iconOnly ? 0 : "0 4px 0" })}
+                      className={css({ margin: iconOnly ? 0 : "0 8px 0 4px" })}
```

Comment:
While we are in here, do we need the `4px` on the left?

Reply:
Thanks for pointing this out, @Jephuff. I just noticed that there is a bit of a discrepancy between the icon spacing in the Space Kit design file:

![image](https://user-images.githubusercontent.com/1319791/98421036-51a82500-203d-11eb-8736-44d535dfad41.png)

...and in our current implementation in Storybook:

![image](https://user-images.githubusercontent.com/1319791/98421066-61c00480-203d-11eb-9c42-bec43c0763d5.png)

What makes this kind of difficult to both notice and address is that in the design file the button uses two words, but in the storybook version it uses just a single short word. So the question is what is the correct spacing?

Visually, I think it looks cleanest to have equal spacing between the icon and the left edge and the icon and the start of the word (or just a hair more between the icon and the button edge, if any).

But I also see that the buttons have a `min-width` applied to them, which makes this a bit tricker. Is there any reasons we need the min width on these?

Regardless of the answer to that, I'm in favor of @Jephuff's suggestion to drop the `4px` on the left. 👍

Reply:
PS - case in point where this came up as awkward was in [this thread for updating the new graph button](https://github.com/mdg-private/studio-ui/pull/3400#discussion_r519037278), and the spacing looks weird:

![image](https://user-images.githubusercontent.com/1319791/98420673-88ca0680-203c-11eb-9088-ba224009f420.png)

Reply:
Thanks for this input! I'll axe that 4px and then send another PR testing getting rid of the min-width!